plugin/grepper.vim

```diff
@@ -690,9 +686,17 @@ function! s:build_cmdline(flags) abort
   let grepprg = s:get_grepprg(a:flags)
 
   if has_key(a:flags, 'buflist')
-    let [shellslash, &shellslash] = [&shellslash, 1]
-    call map(a:flags.buflist, 'shellescape(fnamemodify(v:val, ":."))')
-    let &shellslash = shellslash
+    if has('win32')
+      " cmd.exe does not use single quotes for quoting. Using 'noshellslash'
+      " forces path separators to be backslashes and makes shellescape() using
+      " double quotes. Beforehand escape all backslashes, otherwise \t in
+      " 'dir\test' would be considered a tab etc.
+      let [shellslash, &shellslash] = [&shellslash, 0]
+      call map(a:flags.buflist, 'shellescape(escape(fnamemodify(v:val, ":."), "\\")')
```

Comment:
There is a missing ")" that closes the shellescape function call.

Reply:
Sorry for that. The VM lacks all of my credentials, so I couldn't push to GitHub. In the end I copied the diff by hand to my macbook.. *\*cough\**